src/mantarray_file_manager/files.py

```diff
@@ -118,6 +118,27 @@ def _extract_datetime_from_h5(
     open_h5_file: h5py._hl.files.File,  # pylint: disable=protected-access # WTF pylint...this is a type definition
     metadata_uuid: UUID,
 ) -> datetime.datetime:
+    if str(metadata_uuid) not in open_h5_file.attrs:
```

Comment:
when creating backwards compatibility for files, we want to specifically denote file versions that something is applicable for.  If in the future we accidentally somehow leave out (or purposefully remove) the UTC_BEGINNING_RECORDING_UUID, we don't want this code to execute----unless we've specifically said we want to for that version.  Typically this is done by something at the beginning saying "if version<X, then Y, else Z"